README.md

```diff
@@ -1,5 +1,49 @@
-# injected
-Source code of DLLs / assemblies for injection into target process
+# injectdll
 
-usage:
-py -2.7 example_injector.py \<Process name\> \<path to dll\>
+injectdll is a Python package that allows you to inject DLLs into running processes on Windows (32 or 64-bit).
+
+This can be useful for various tasks, such as debugging or adding functionality to existing applications.
+
+It contains test DLLs for classic Win32 API and WPF applications.
+
+Link to PyPi: <https://pypi.org/project/injectdll/>
+
+## How to use
+
+    python.exe example_injector.py \<Process name\> \<path to dll\> 
+
+## Development
+
+ 1. Clone repository: <https://github.com/pywinauto/injected.git>
+ 2. Create new features or improve exiting (for the injectdll package)
```

Comment:
typo: exiting -> existing, but I would completely remove all "Development" steps or write "Issues and pull requests are welcome!"

Reply:
Ok, will be fixed in next commits